scout/server/blueprints/variants/forms.py

```diff
@@ -153,9 +154,30 @@ class StrFiltersForm(FlaskForm):
     """docstring for StrFiltersForm"""
 
     variant_type = HiddenField(default="clinical")
-    chrom = TextField("Chromosome")
+
+    hgnc_symbols = TagListField("HGNC Symbols/Ids (case sensitive)")
     gene_panels = SelectMultipleField(choices=[])
-    repids = TagListField()
+
+    chrom = SelectField(
+        "Chromosome", [validators.Optional()], choices=CHROMOSOME_OPTIONS, default=""
+    )
+
+    start = IntegerField("Start position", [validators.Optional()])
```

Comment:
You could skip adding many (if not all) of these fields if you make StrFiltersForm a subclass of VariantFiltersForm instead of FlaskForm. This way it can inherit:
- chrom
- start
- end
- cytoband_start
- cytoband_end
- All the filters block
- hide dismissed
-  export

Reply:
Hm, it has some extra stuff that I would rather not include but I could make another least common denominator class and have them all inherit again.

Reply:
but then you just don't include those in the html page and they won't be a problem

Reply:
I think

Reply:
Meh, I give in - changing and going for another LCD will require loads of extra testing. I agree and will just inherit - those extra fields should not hurt anyone, and ideally unique ones should only be on the child forms eventually.

Reply:
One easy test that you could do is checking the request form after it arrives to the view, I'm pretty sure that it won't contain the extra keys if you don't include them in the html page